.github/workflows/pr-reviewed-trigger.yml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Pull Request Reviewed
+
+on:
+  pull_request_review:
```

Comment:
This event brings a interesting race condition:

1. Approve the Pull Request (PR):
    - This action triggers the "Pull Request Reviewed" workflow.
    - [Example Run](https://github.com/apache/kafka/actions/runs/12329199776/job/34413629998)

2. Add ci-approve:
    - This action attempts to locate the "pending" run_id.
    - it mistakenly retrieves the run_id of the "Pull Request Reviewed" workflow, resulting in an error.
[Error Example](https://github.com/apache/kafka/actions/runs/12329250710/job/34413372498?pr=18184)

Perhaps the issue is invalid, as pull requests should not be approved before the CI process completes. :smile:

Reply:
Filed #18200 to (hopefully) fix this